build.cmd

```diff
@@ -10,25 +13,34 @@ IF "%*"=="" (
   SET SCRIPTARGS=%*
 )
 
-IF EXIST %CACHED_NUGET% goto copynuget 
-echo Downloading latest version of NuGet.exe... 
-IF NOT EXIST %LocalAppData%\NuGet md %LocalAppData%\NuGet 
-@powershell -NoProfile -ExecutionPolicy unrestricted -Command "$ProgressPreference = 'SilentlyContinue'; Invoke-WebRequest 'https://www.nuget.org/nuget.exe' -OutFile '%CACHED_NUGET%'" 
- 
-:copynuget 
-IF EXIST .nuget\nuget.exe goto restore 
-md .nuget 
-copy %CACHED_NUGET% .nuget\nuget.exe > nul 
- 
-:restore 
-IF EXIST packages\KoreBuild goto run  
-.nuget\NuGet.exe install KoreBuild -ExcludeVersion -o packages -nocache -pre -Source https://myget.org/F/aspnetvnext
```

Comment:
Not sure about removing the `-Source` argument here. The change won't affect CI builds because earlier steps create a `NuGet.config` file containing the Right Stuff:tm:.

@joeloff any expected issues for local builds i.e. to most people have reasonable default feeds, containing `https://myget.org/F/aspnetvnext` or newer? I'll add a `NuGet.config` file if you think it's worthwhile.